Integrations/integration-Jask_v2.yml

```diff
@@ -0,0 +1,1540 @@
+            arg_list = argToList(demisto.getArg(translate[v]))
+            if len(arg_list) == 1:
+                q += " {}:{}".format(v, arg_list[0])
+            elif len(arg_list) > 1:
+                q += " {}:{}".format(v, ','.join(arg_list))
+        return q
+
+
+    def add_status_to_q(q, status):
+        """
+        Convert the status value for the query string
+        """
+        if status != 'not closed':
+            q += " status:{}".format(status)
+            return q
+
+        return q + " -status:closed"
+
+    def add_time_to_q(q, time_field, relative_time=None, time_from=None, time_to=None):
+        """
+        Add the time filter to the query string for created or lastSeen
+        time_field_types = ['created', 'lastSeen']
+        Defaults to All time if no fields specified
+        """
+        if relative_time:
+            if time_from or time_to:
+                return_error('You cannot specify absolute times [time-to, time-from] ' +
+                    'with relative time [{}]'.format(time_field))
+            else:
+                if translate_time_query(relative_time, time_field) != '':
+                    q += translate_time_query(relative_time, time_field)
+        elif time_from and time_to:
+            q += " {}:{}..{}".format(time_field, convert_string_date_to_date(time_from), convert_string_date_to_date(time_to))
+        elif time_from or time_to:
+            return_error('You must specify both absolute times [time-to, time-from] ' +
+                'or relative time [{}]'.format(time_field))
+
+        return q
+
+
+    def get_insight_from_context(context):
+        """
+        Looks for an Insight in the Jask.Insight context key, so we can update status, or add comments
+        If there are more then one, print the list of Insights to be helpful.
+        """
+        if type(context) == dict:
+             return context.get('Id')
+        else:
+            error_md = tableToMarkdown("Too many Insights on this Incident, please supply Insight ID to command instead:",
+                                       context, ['Id', 'Name'])
+            demisto.results({
+                'Type': entryTypes['error'],
+                'HumanReadable': error_md,
+                'ContentsFormat': formats['json'],
+                'Contents': context
+            })
+            sys.exit(0)
+
+
+    ''' COMMAND FUNCTIONS '''
+    def get_insight_details_command():
+        """
+        Get insight details, basically print everything about the insight to war room
+        """
+        # get the insight details from the jask api, and start building our results
+        insight_id = demisto.getArg('insight-id')
+        path = "insights/{}".format(insight_id)
+        resp_json = jask_api_request('GET', path)
+
+        # error check
+        if len(resp_json['errors']) > 0:
+            jask_return_error(resp_json['errors'])
+
+        insight_resp = resp_json['data'] or "none"
```

Comment:
No need for this, you can just use condition below to check `if resp_json.get('data')`